--- ale-0.6.0/Makefile.orig	2004-01-27 06:15:06.000000000 +0100
+++ ale-0.6.0/Makefile	2004-05-21 21:18:17.000000000 +0200
@@ -68,7 +68,7 @@
 	rm -f ale
 
 ale-phony: ale.cc d2.cc *.h d2/*.h d2/render/*.h d2/render/psf/*.h
-	g++ -o ale $(CFLAGS) ale.cc d2.cc $(LDFLAGS)
+	$(CXX) -o ale $(CFLAGS) ale.cc d2.cc $(LDFLAGS)
 
 # The following approach to building a Windows binary is probably very
 # dependent on the host platform configuration.  The above target may be a
